### PR TITLE
Dynamic Stack Bifurcation

### PR DESCRIPTION
**Pull request by lsgunth**
This PR adds the stack-bif command to dynamically change the stack bifurcation of an unbound port.

The code is pretty straightforward implementation of MRPC commands, although the command line for specifying the bifurcation of a stack is a bit complicated.

### CODE REVIEW

**Comment by lsgunth** [comment]
Any progress on reviewing this and getting it merged?

File: lib/switchtec.c [hunk 1]
```diff
@@ -1963,4 +1972,123 @@ int switchtec_calc_lane_mask(struct switchtec_dev *dev, int phys_port_id,
...
+	if (dev->gen == SWITCHTEC_GEN4)
+		return stack_id * 8 + port_id < 52;
+
+	return true;
+}
+
+/**
+ * @brief Return the number of stack ports used for a given bifurcation
+ * @param[in] dev	Switchtec device handle
+ * @param[in] stack_id	Stack ID t
```
Comment (kelvin-cao) [inline lib/switchtec.c:1994]: Stack ID t -> Stack ID

Comment (lsgunth) [reply lib/switchtec.c]: Fixed that up and rebased onto devel.

**Review by kelvin-cao — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**